docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -66,5 +75,15 @@ To check if you have this driver installed:
 4. Click the **Driver** tab to see the driver version. 
    ![](../img/DeviceDriverVersion.PNG)
 
-You are done updating your device driver and are ready to use your GPU. Proceed to the <a href="openvino_docs_install_guides_installing_openvino_windows.html#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
+You are done updating your device driver and are ready to use your GPU. Now you are ready to try out the toolkit.
```

Comment:
These two parts are the same. Can we make it a dedicated section as something like "Next Steps"?

Reply:
```suggestion
You are done updating your device driver and are ready to use your GPU.

## What’s Next?
```

@xu-yuan1 maybe something like this? I used "What's Next?" instead since we are using the same pharse in other installation guides.

Reply:
Sounds good. Maybe we can keep the "Now you are ready to try out the toolkit." intro sentence in the new section. :)

Reply:
I thought about this too, but won't it sound a little bit like a repetition of "... ready to use your GPU"? What If we change it to be like this:

```suggestion
You are done updating your device driver and are ready to use your GPU.

## What’s Next?

You can try tryout the toolkit with:
```

Reply:
Sounds good.